pwndbg/gdblib/config.py

```diff
@@ -66,6 +79,15 @@ def get_set_string(self) -> str:
             # Anyway, to avoid some unexpected behaviors, we'll still set `self.param.value` to 0 here.
             self.param.value = 0
         else:
+            if self._custom_param_class:
+                if (self._custom_param_class == "PARAM_ZUINTEGER" and self.value < 0) or (  # type: ignore
+                    self._custom_param_class == "PARAM_ZUINTEGER_UNLIMITED" and self.value < -1  # type: ignore
```

Comment:
Since this is making more and more complex and less and less clear, shall we maybe do something along the lines of:

```py
def __get_set_string_gdb_le_9(self): ...
def __get_set_string_gdb_gte_9(self): ...

if gdb_version >= 9:
    get_set_string = __get_set_string_gdb_gte_9
else:
    get_set_string = __get_set_string_gdb_le_9
```
@gsingh93 WDYT?

Reply:
+1, I think that'll be less complicated and cleaner.